app/views/docs/index.html.md

```diff
@@ -1,5 +1,19 @@
 # API
 
+## i18n
+
+All API endpoints allows a `locale` parameter to specify the language or locale set to use. This affects the data returned and how to make searches: for example if English or Amharic versions of services' name should be used.
```

Comment:
The wording after "for example..." is hard to follow.
Maybe something like "for example, to specify whether to use English or Amharic names for a facility request the following parameters can be used"?